PROVESFlightControllerReference/test/int/0_radio_test.py

```
@@ -0,0 +1,57 @@
+    proves_send_and_assert_command(
+        fprime_test_api,
+        f"{lora}.TRANSMIT",
+        ["ENABLED"],
+    )
+
+
+@pytest.mark.rf_unsafe
+def test_01_transmit_enabled(fprime_test_api: IntegrationTestAPI, start_gds):
+    """Enabling transmit must not produce any LoRa error/warning events."""
+    start: TimeType = TimeType().set_datetime(
+        datetime.now(), time_base=TimeType.TimeBase("TB_DONT_CARE")
```

Ruff (0.15.12)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)